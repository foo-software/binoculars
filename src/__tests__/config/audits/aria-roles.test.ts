import auditAriaRoles from '../../../config/audits/aria-roles';

describe('aria-roles audit', () => {
  it('should match snapshot', () => {
    expect(auditAriaRoles('en')).toMatchSnapshot();
  });
});
