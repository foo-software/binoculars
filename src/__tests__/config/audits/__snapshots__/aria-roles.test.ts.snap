// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aria-roles audit should match snapshot 1`] = `
Object {
  "description": "ARIA roles must have valid values in order to perform their intended accessibility functions. [Learn more](https://web.dev/aria-roles/).",
  "title": "\`[role]\` values are valid",
}
`;
